SPECS-SIGNED/mlnx-ofa_kernel-modules-signed/mlnx-ofa_kernel-modules-signed.spec

```diff
@@ -44,7 +45,7 @@
 Summary:	 Infiniband HCA Driver
 Name:		 %{_name}-signed
 Version:	 24.10
-Release:	 16%{?dist}
+Release:	 17%{release_suffix}%{?dist}
```

Comment:
Is this going to fix the pinning issue for the cuda spec? I think we end up needing to manually edit that package constantly with an explicit version?

Reply:
my understanding being that, each time kernel is upgraded, the toolkit will always rebuild the OOT specs because their release nb integrates kernel version/release. 
Cuda might be a little different since it belongs to proprietary repo